lms/djangoapps/courseware/module_render.py

```diff
@@ -194,14 +194,13 @@ def toc_for_course(user, request, course, active_chapter, active_section, field_
                     }
 
                     #
-                    # Add in rendering context for proctored exams
-                    # if applicable
+                    # Add in rendering context if exam is a timed exam (which includes proctored)
                     #
-                    is_proctored_enabled = (
-                        getattr(section, 'is_proctored_enabled', False) and
-                        settings.FEATURES.get('ENABLE_PROCTORED_EXAMS', False)
+
+                    section_is_time_limited = (
+                        getattr(section, 'is_time_limited', False) and settings.FEATURES['ENABLE_SPECIAL_EXAMS']
```

Comment:
Usually settings.FEATURES.get('ENABLE_PROCTORED_EXAMS', False) is used to provide a default value if necessary.